cmd/dex-worker/main.go

```diff
@@ -51,6 +51,9 @@ func main() {
 
 	enableClientRegistration := fs.Bool("enable-client-registration", false, "Allow dynamic registration of clients")
 
+	// Client credentials administration
+	enableClientUserCreation := fs.Bool("enable-client-user-creation", false, "allow admin clients to create users using client-credentials flow")
```

Comment:
I'd prefer this was a hard choice. e.g. Turning this on forced a user to only use client creds (and not an ID Token).